frappe/desk/form/utils.py

```diff
@@ -63,7 +62,7 @@ def add_comment(reference_doctype, reference_name, content, comment_email):
 		doctype = 'Comment',
 		reference_doctype = reference_doctype,
 		reference_name = reference_name,
-		content = sanitize_html(content),
+		content = content,
```

Comment:
what about sanitization?

Reply:
- sanitization removes the image source which leads to images not being rendered.
- fieldtype of `content` has been changed from HTML to Text Editor to facilitate this.